services/bepro-service.ts

```diff
@@ -191,6 +191,12 @@ class BeproFacet {
     return network.getOraclesResume(this.address);
   }
 
+  async getSettlerTokenAllowance(networkAddress: string = publicRuntimeConfig?.contract?.address): Promise<number> {
+    const network = await this.getNetworkObj(networkAddress);
+    
+    return this.getAllowance(network.settlerToken.contractAddress, this.address, networkAddress);
```

Comment:
_please_, for everyones' sanity, don't call a function that was not declared yet:

```
function B() {
  A(); // I don't know what A does, as it has not been declared on the code yet
}

function C() {
  A() // Same
}

function A() {} // I needed to get to the end of the file so I can read this declaration and now scroll back up to make sense of the logic inside the functions
```

but if you declare `A()` first, then everything falls into place :)